docs/docs/deploy/deploy-rasa-plus.mdx

```diff
@@ -77,6 +43,7 @@ Create the pull secret by running the corresponding command for your cluster env
 kubectl --namespace <your namespace> \
     create secret docker-registry rasa-plus-pull-secret \
     --docker-server=gcr.io \
+    --docker-email=test-customer@rasa-releases.iam.gserviceaccount.com \
```

Comment:
should this be the same as https://github.com/RasaHQ/rasa/pull/11654/files#diff-d4cb3748ccc8d1489ff62d746835becb7134eb01d10b713059476f354b054c4fR30 ? should we mention that this should be set to the correct value?

Reply:
i can definitely add a comment, not sure about the link, it just sends me back to the the files tab in this PR?

Reply:
i tried to link to a line earlier in this file, probably doesn't work anymore because the file changed 🤷 just look a couple lines earlier in that docs file (you need to expand in the diff). we have an example `json` in there fore a service account, that account uses a different email. my assumption would be that these two need to be the same

Reply:
Yep, I found it and amended, but to double-check with @camattin if this assumption is correct: does the docker email need to match the client email in the `json` example?